packages/pf-codemods/README.md

```diff
@@ -638,7 +639,23 @@ Out:
   }}
 />
 ```
+### popper-remove-popperMatchesTriggerWidth [(#8724)](https://github.com/patternfly/patternfly-react/pull/8724)
+
+We've removed the `popperMatchesTriggerWidth` prop from Popper, minWidth, maxWidth and width props now can be used to modify the popper width.
```

Comment:
```suggestion
We've removed the `popperMatchesTriggerWidth` prop from Popper. `minWidth`, `maxWidth`, and `width` props can instead be used to modify the Popper width.
```

Reply:
Thanks for the feedback, I've made the changes.

Reply:
Thank you! And awesome job on this 🔥